src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsassessforearlyreleaseapi/service/prison/AddressService.kt

```diff
@@ -109,6 +110,7 @@ class AddressService(
   ): CasCheckRequestSummary {
     val offender = offenderRepository.findByPrisonNumber(prisonNumber)
       ?: error("Cannot find offender with prisonNumber $prisonNumber")
+    val assessmentEntity = assessmentService.getCurrentAssessment(prisonNumber)
```

Comment:
don't need to load here.. can get assessment from `offender.currentAssessment()`